frontier-chain/node/Cargo.toml

```diff
@@ -80,6 +80,9 @@ fp-evm = { workspace = true, features = ["default"] }
 fp-rpc = { workspace = true, features = ["default"] }
 frontier-template-runtime = { workspace = true, features = ["std"] }
 
+# Primitive types
+frontier-primitives = { workspace = true }
```

Comment:
this doesn't need to here, imo. the usual practice is to re-export types from the runtime